core/storage/question/gae_models.py

```diff
@@ -103,3 +103,45 @@ def create(
             language_code=language_code)
 
         return question_model_instance
+
+
+class QuestionSkillLinkModel(base_models.VersionedModel):
+    """Model for storing Question-Skill Links.
+
+    The ID of instances of this class has the form
+    {{random_hash_of_16_chars}}
+    """
+    ALLOW_REVERT = True
+
+    # The ID of the question.
+    question_id = ndb.StringProperty(required=True, indexed=True)
+    # The ID of the skill to which the question is linked.
+    skill_id = ndb.StringProperty(required=True, indexed=True)
+    # Status of the review: Pending(0)/Approved(1)/Rejected(-1)
+    review_status = ndb.IntegerProperty(required=True, choices=[-1, 0, 1])
```

Comment:
Let's drop review_status, here and elsewhere.

Reply:
Removed review status & difficulty as per discussion